media-gfx/apng2gif/apng2gif-1.7.recipe

```diff
@@ -0,0 +1,45 @@
+SUMMARY="Converts APNG animations into GIF format"
+DESCRIPTION="This program converts APNG animations into animated GIF format. \
+It performs the reverse of gif2apng."
+HOMEPAGE="http://apng2gif.sourceforge.net/"
+COPYRIGHT="2009-2015 Max Stepin"
+LICENSE="Zlib"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/apng2gif/apng2gif-$portVersion-src.zip"
+CHECKSUM_SHA256="6c7b2cce851378f2fc8f6b490ce098e9cf0038aa05da7099771c8be94620e6ca"
+SOURCE_DIR=""
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	apng2gif$secondaryArchSuffix = $portVersion
+	cmd:apng2gif = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libz$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:ld$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	mkdir -p $prefix/bin
+	cp apng2gif $prefix/bin
```

Comment:
Could you please use the install command, instead of mkdir and cp?
We have a recipe which you can get inspiration from:
app-misc/hexcompare/hexcompare-1.0.4.recipe